### PR TITLE
fix: Ensure `_UnoGeneratePriMarker` is invoked before `_UnoAssetsGetCopyToPublishDirectory`

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

### <samp>🤖 Generated by Copilot at 42a098f</samp>

Fix a bug where assets were not published with the app bundle by changing the order of MSBuild targets in `uno.ui.tasks.assets.targets`.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets [hunk 1]
```diff
@@ -198,7 +198,7 @@
 	as nuget packages content.
 	-->
 	<Target Name="_UnoAssetsGetCopyToPublishDirectory"
-	BeforeTargets="GetCopyToPublishDirectoryItems">
+			BeforeTargets="GetCopyToPublishDirectoryItems;_UnoGeneratePriMarker">
```
Comment (ajpinedam) [inline src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets:201]: Just for personal knowledge: what's the purpose/benefit of this change?

Comment (Youssef1313) [reply src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets:201]: @ajpinedam Trying to unblock toolkit build.

It fails with `Error NU5019: File not found: 'D:\a\1\s\src\library\Uno.Toolkit.Cupertino\bin\Uno.Toolkit.UI.Cupertino\Release\net7.0-ios\Uno.Toolkit.UI.Cupertino.uprimarker'.`

What happens there is that `_UnoGeneratePriMarker` was run *after* `_UnoAssetsGetCopyToPublishDirectory`, and so uprimarker files wasn't copied.

I'm not yet sure what is special about the toolkit build though.

**Comment by Youssef1313** [comment]
Draft as I'm suspecting this to be a workaround for some problem in toolkit build.

**Comment by Youssef1313** [comment]
This broke template tests 😕

**Comment by Youssef1313** [comment]
This change looks wrong actually. Digging more.

**Pull request closed without merging**